modules/oe_theme_content_entity_organisation/oe_theme_content_entity_organisation.module

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @file
+ * Module file is used for theming the Organisation entity.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Url;
+use Drupal\Component\Utility\UrlHelper;
+use Drupal\file\FileInterface;
+use Drupal\media\MediaInterface;
+use Drupal\oe_theme\ValueObject\ImageValueObject;
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_content_entity_organisation_preprocess_oe_organisation(&$variables): void {
+  $entity = $variables['entity'];
+  $fields_values = ['name' => 'name', 'oe_acronym' => 'acronym'];
+  foreach ($fields_values as $entity_field => $pattern_field) {
+    if (!$entity->get($entity_field)->isEmpty()) {
+      $variables[$pattern_field] = $entity->get($entity_field)->value;
+    }
+  }
+
+  // Extract the image if Media and File entities exist.
+  $media_image = $entity->get('oe_logo')->entity;
+  if ($media_image instanceof MediaInterface) {
+    $media_image = \Drupal::service('entity.repository')->getTranslationFromContext($media_image);
+    $cacheability = CacheableMetadata::createFromRenderArray($variables);
+    $cacheability->addCacheableDependency($media_image);
+    $image_item = $media_image->get('oe_media_image')->first();
+    $file = $image_item->get('entity')->getValue();
+
+    if ($file instanceof FileInterface) {
+      $variables['logo'] = ImageValueObject::fromImageItem($image_item);
+      $cacheability->addCacheableDependency($file);
+    }
+    $cacheability->applyTo($variables);
+  }
+
+  // Set contact page url.
+  $variables['contact_page_url'] = '';
+  if (!$entity->get('oe_contact_url')->isEmpty()) {
+    $link = $entity->get('oe_contact_url')->first()->getValue();
+    if (UrlHelper::isExternal($link['uri'])) {
+      $variables['contact_page_url'] = $link['uri'];
+    }
+    else {
+      $variables['contact_page_url'] = Url::fromUri($link['uri'])->toString();
+    }
+  }
```

Comment:
Let's replace the above with this, as the field object can already access the URL object and its toString method:

```php
  if (!$entity->get('oe_contact_url')->isEmpty()) {
    /** @var \Drupal\link\Plugin\Field\FieldType\LinkItem $link */
    $link = $entity->get('oe_contact_url')->first();
    $variables['contact_page_url'] = $link->getUrl()->toString();
  }
```